cmd/cmd.go

```diff
@@ -1473,6 +1473,8 @@ func NewCLI() *cobra.Command {
 				envVars["OLLAMA_LLM_LIBRARY"],
 				envVars["OLLAMA_GPU_OVERHEAD"],
 				envVars["OLLAMA_LOAD_TIMEOUT"],
+				envVars["OLLAMA_DRAFT_GPU_DEVS"],
```

Comment:
This is entirely a personal opinion, but would it perhaps be more uniform to spell out DEVICES rather than using DEVS? The word DEVICES does not seem to be shorthanded to DEVS in the ollama repo, and I find the extra few characters helpful in making it explicit that this is about devices rather than development or something else.
My suggestion would apply to every use of the shorthand "devs" even in your variable names.

Of course, this is only a suggestion.

Reply:
I agree - DEVICES is the better name! However, in an attempt to fix https://github.com/ollama/ollama/pull/8134#discussion_r1890840858 I have remove these new `OLLAMA_DRAFT_*` environment variables again (which also resolves the naming issue). FIX: https://github.com/ollama/ollama/pull/8134/commits/59e036b6b0e3b2674c27c7d57aae0f7d529c0f31